components/automate-chef-io/content/docs/applications-setup.md

```diff
@@ -0,0 +1,218 @@
+
+For example:
+
+```shell
+HAB_FEAT_EVENT_STREAM=1
+hab sup run \
+  --event-stream-application="AmazingEnterpriseApp" \
+  --event-stream-environment="acceptance" \
+  --event-stream-site="us-west-2" \
+  --event-stream-url="automate.company.com:4222" \
+  --event-stream-token="1234567890abcdefgh"
+```
+
+## Authorize Communication from Chef Habitat to Chef Automate
+
+The Transport Layer Security (TLS) protocol provides privacy and data integrity during communication between Chef Automate and Chef Habitat. Provide Chef Habitat with Chef Automate's TLS certificate to authorize communication between the two.
+
+### Retrieve Chef Automate's TLS key
+
+Chef Automate's TLS certificate file is located at `/hab/svc/automate-load-balancer/data/<servername>.crt`.
+
+From Chef Automate:
+
+1. On the command line, run `chef-automate config show`
+1. Copy the `[[global.1.frontend_tls]]` certificate contents from the certificate list
+1. Make sure to copy it entirely, including `-----BEGIN CERTIFICATE...END CERTIFICATE-----`
+1. Save the file as `automate.cert` somewhere safe and accessible to you
+
+### Share the TLS key with Chef Habitat
+
+Share the automate certificate with the Chef Habitat supervisor.
+
+Use **one** of these three options:
+
+#### Copy the Chef Automate Certificate into the `/hab/cache/ssl` Directory
+
+Chef Habitat automatically searches that directory and uses the certificate at start up.
+
+  * On Linux systems, copy the cert to`/hab/cache/ssl` if you are the root user and `~/.hab/cache/ssl` if you are a non-root user. If you are a non-root user, the full file path is `/Users/username/.hab/cache/ssl/automate.cert`, located in your local user's home directory.
+  * On Windows systems, store your certs in in `C:\hab\cache\ssl`
+
+#### Add the TLS certificate to your event stream command
+
+Pass the certificate file as a parameter by adding it to your event stream command:
+
+```shell
+--event-stream-server-certificate=/path/to/automate.cert
+```
+
+For example:
+
+```shell
+HAB_FEAT_EVENT_STREAM=1 hab sup run \
+--event-stream-application=MY_APP \
+--event-stream-environment=MY_ENV \
+--event-stream-site=MY_SITE \
+--event-stream-url=AUTOMATE_HOSTNAME:4222 \
+--event-stream-token=API_TOKEN
+--event-stream-server-certificate=/path/to/automate.cert
+```
+
+#### Add the TLS certificate to your certificate store
+
+Add the certificate to your systems platform-specific certificate store.
+These are: `SChannel` on Windows, `Secure Transport` on OSX, and `OpenSSL` on all other platforms. Please follow the instructions for your specific operating system.
+
+## Troubleshooting
+
+### Create a new TLS key pair
+
+You can change Chef Automate's automate front-end TLS key pair by following the [load balancing configuration](https://automate.chef.io/docs/configuration/#load-balancer-certificate-and-private-key) documentation.
+
+### Re-Enable TLS on Chef Automate
+
+In some cases, front-end TLS communication may be previously disabled.
```

Comment:
disabled for the event gateway service only. We should disambiguate that this does not mean we disabled TLS for automate, just the habitat event stream.